prow/cmd/sinker/main.go

```diff
@@ -138,13 +138,38 @@ func main() {
 }
 
 type controller struct {
-	logger        *logrus.Entry
-	prowJobClient prowv1.ProwJobInterface
-	podClients    []corev1.PodInterface
-	config        config.Getter
+	logger         *logrus.Entry
+	prowJobClient  prowv1.ProwJobInterface
+	podClients     []corev1.PodInterface
+	config         config.Getter
+	currentMetrics *sinkerReconciliationMetrics
+}
+
+type sinkerReconciliationMetrics struct {
```

Comment:
nit: We're inside of `prow` here so everything we do is "by Prow" -- we can name without that suffix, `podsRemoved` for instance

Reply:
Done